packages/lib/server/repository/profile.ts

```diff
@@ -249,20 +250,68 @@ export class ProfileRepository {
     });
   }
 
+  static async createManyForExistingUsers({
```

Comment:
We use this method in the `handleGroupEvent` so we can `createMany` profiles for existing users rather than making individual DB calls.